pyproject.toml

```diff
@@ -257,22 +265,15 @@ ignore = [
 ]
 
 external = ["V"]
-target-version = "py38"
-force-exclude = true
-line-length = 88
-output-format = "grouped"
 ignore-init-module-imports = true
-show-source = true
```

Comment:
The ruff docs state that [show-source](https://docs.astral.sh/ruff/settings/#show-source) is deprecated.

You can get this behavior by setting output_format to full. What do you think?